### ISSUE TITLE
Lineup with original API calls

### ISSUE THREAD

**Issue by WallyCZ** [PoGo-Devs/PoGo-API#1]
### **Actual behavior:**

Current API calls are different from original client. Main difference is, that some type of messages are not pushed to server directly but only with other messages. These messages comes with every request:
GetHatchedEggs
GetInventory
CheckAwardedBadges
DownloadSettings

GetInventory sends only deltas from last request (we are requesting always full inventory)
DownloadSettings use hash and is sent back to client only if hash (content) changed
### **Expected behavior:**

Change RPCs methods and GameClient so we match with original API calls. This will have probably impact on how are currently these messages handled. We must check in one place responses and sends some events to other consumers who waits on these changes (f.e. there must be hook for player stats change, inventory item change and so on)
### **Steps to reproduce the problem:**

Compare our comm. log with original comm log. Sequence of messages should be same.

**Comment by robertmclaws** [comment]
So let me make sure I understand: If I send a request to Client.Fort.SearchFort(), I might get back a FortDetailsResponse, a GetInventoryResponse, and a CheckAwardedBadgesResponse in the same payload?

**Comment by WallyCZ** [comment]
Exactly. And GetInventory contains deltas of inventory, player data, pokemon, ....

**Comment by Migsi** [comment]
I'm sorry if I ask here, but will this repo contain the API used by PoGo or is this repo made for an API to access PoGo? The description of the repo seems a bit unclear, but this thread is talking about the calls to the server, so if somebody could enlighten me, that'd be great! :)

**Comment by robertmclaws** [comment]
This repo is about the PoGo app for Windows and our fork of the API client that powers the app.

**Comment by Migsi** [comment]
@robertmclaws You misunderstood. I know this organization is for the PoGo UWP app ;) I was just intrested if this specific repo (PoGo-API) is going to contain the API or if that will stay in the "PoGo" repo, as it is right now.

**Comment by ST-Apps** [comment]
This will just contains issues related to game API, no code.

**Comment by Migsi** [comment]
Question answered, thx!
